src/App.jsx

```diff
@@ -1,9 +1,39 @@
 import React from "react";
+//routing
+import { Routes, Route, Navigate, useLocation } from "react-router-dom";
+import { ROOT, MESSAGES, MAIN_CATEGORY } from "@routes/routeConstants.js";
+//components
+//TODO: import NAVBAR
+import Messages from "./pages/Messages.jsx";
 
 const App = () => {
   return (
     <>
-      <h1 className="text-4xl font-bold">React Boilerplate</h1>
+      <div className="h-[64px] bg-gray-500">NAVBAR COMPONENT</div>
```

Comment:
Is this title necessary for your component? If it is, please include it directly within the component where it will be used, rather than placing it in the App.js

Reply:
This is a placeholder, will be replaced by a navbar component that amir is working on